app/zapier/files/upload.js

```diff
@@ -0,0 +1,132 @@
+
+//
+// (c) Copyright Ascensio System SIA 2023
+//
+
+// @ts-check
+
+const contentDisposition = require("content-disposition")
+const FormData = require("form-data")
+
+/**
+ * @typedef {import("../../docspase/files/files.js").ChunkData} ChunkData
+ * @typedef {import("../../zapier/files/actions.js").UploadFileFields} UploadFileFields
+ */
+
+/**
+ * @typedef {Object} ErrorData
+ * @property {string} message
+ * @property {string} stack
+ */
+
+/**
+ * @typedef {Object} UploadFileData
+ * @property {number} id
+ * @property {number} folderId
+ * @property {number} version
+ * @property {string} title
+ * @property {boolean} uploaded
+ * @property {FileData} file
+ */
+
+/**
+ * @typedef {Object} Headers
+ * @property {string} fileName
+ * @property {number} fileSize
+ */
+
+/**
+ * @typedef {import("./actions.js").FileData} FileData
+ */
+
+class Upload {

```

Comment:
Using "upload" as a verb to name a class isn't a good idea. It would be better to rename it to something like "Uploader" for clarity.

Reply:
> Uploader

Renamed